api/apps/api/src/modules/scenarios/locks/scenario.lock.api.entity.ts

```diff
@@ -22,3 +23,8 @@ export class ScenarioLockEntity {
   })
   createdAt!: Date;
 }
+
+export class ScenarioLockResult {
+  @ApiProperty()
+  data!: ScenarioLockEntity;
+}
```

Comment:
I am not sure I get this - are you shaping the reponse as JSON:API? If so, that's great - but you'd need some more properties, and I would then assert more in the test method `ThenScenarioLockInfoIsReturned()` - at least that `data.type` matches what would be the JSON:API type for these lock response payloads.

If this is for something else, then ignore me (but I would not be clear what it is for.

If it's the JSON:API, have a look at other examples in bits such as scenario entity: examples from CRUD parts can help with this (or ask me if you get stuck).